core/internals.h

```diff
@@ -74,8 +76,11 @@
 #define URI_REGISTRATION_SEGMENT_LEN    2
 #define URI_BOOTSTRAP_SEGMENT           "bs"
 #define URI_BOOTSTRAP_SEGMENT_LEN       2
+#define URI_DELETE_ALL_SEGMENT          "/"
+#define URI_DELETE_ALL_SEGMENT_LEN      1
```

Comment:
I even didn't understand the LWM2M bootstrap interface in this issue "delete all".
The bootstrap interface doesn't provide read function, so the only option to get to a defined point is to delete all. So I don't see, why this "delete all" is left to the bootstrap serve instead of being mandatory for a bootstrap.